File: Calculate BMI.py
def bmi(weight, height):
    #your code here
    result = weight/(height**2)
    
    if(result <= 18.2):
        return "Underweight"
    elif(result <= 25.0):
        return "Normal"
    elif(result <= 30.0):
        return "Overweight"
    elif(result > 30.0):
        return "Obese"

File: Calculate BMI.js
function bmi(weight, height) {
  var x = (weight/(height**2));
  
  if( x <= 18.5){
    return "Underweight"
  } else if(x <= 25.0){
    return "Normal"
  } else if(x <= 30.0){
    return "Overweight"
  } else if(x > 30){
    return "Obese"
  }
}
